### ISSUE TITLE
Going from plot => command

### ISSUE THREAD

**Issue by berrytj** [quantifind/wisp#19]
I love the stack nature of wisp, but when I've been adding and deleting plots in rapid succession, sometimes I can't figure out which scala commands a plot represents.  There are at least two ways to ameliorate this:
1) Tag series with their result val (e.g. `res15`).
2) Tag series with creation information.  For example, see the Incanter plot below.  This is obviously more helpful in the (iterable, function) case than the (iterable, iterable) case.

Thoughts?

![screen shot 2015-02-03 at 9 11 29 am](https://cloud.githubusercontent.com/assets/1096006/6025095/ad08cfe6-ab84-11e4-8a61-c7a5565eeb60.png)

**Comment by AustinBGibbons** [comment]
Agreed. I've started doing this for some of the other things I've been looking at - here I tag probability distributions with the distribution/parameters:

![chart 11](https://cloud.githubusercontent.com/assets/1144402/6026204/af838be0-ab8d-11e4-81f6-40cd5f3f6570.png)

I'm not totally sure how to accomplish this generally, perhaps prefer (2) and fall back to (1)? But I'm also not sure how to implement that functionality, so I may put this one on the back-burner

**Comment by berrytj** [comment]
Cool, well as long as you think it's a good idea, I may try out an implementation

**Comment by AustinBGibbons** [comment]
Definitely! Also is your use case multiple series on the same plot, or multiple plots with 1 series each?

I've been working with the mindset of using the legend to inform the plot, but I like the idea of auto-populating the axes titles as well (but that might get convoluted with multiple series)

**Comment by berrytj** [comment]
Agreed – auto-populating the axis titles is nice, but supporting multiple use cases via the legend is a more flexible/powerful solution.  If I dig into this I'll think about the relationship between them.

**Comment by berrytj** [comment]
s/multiple use cases/multiple series/
